alb.tf

```diff
@@ -1,11 +1,11 @@
 resource "aws_alb" "eq" {
```

Comment:
if i'm bring really fussy we should remove the eq from the names

Reply:
I think it needs some name, what should we call it instead `default`/`main`?